test/test.hs

```diff
@@ -581,8 +581,8 @@ tests = testGroup "hevm"
               }
              }
             |]
-        (_, [Cex (_, ctr)]) <- withSolvers Z3 1 Nothing $ \s -> checkAssert s [0x32] c (Just ("fun(uint8)", [AbiUIntType 8])) [] defaultVeriOpts
-        assertBool "Access must be beyond element 2" $ (getVar ctr "arg1") > 1
+        (_, [Cex (_, _)]) <- withSolvers Z3 1 Nothing $ \s -> checkAssert s [0x32] c (Just ("fun(uint8)", [AbiUIntType 8])) [] debugVeriOpts
+        -- assertBool "Access must be beyond element 2" $ (getVar ctr "arg1") > 1
```

Comment:
why is this commented out?

Reply:
It is because of this [issue](https://github.com/ethereum/hevm/issues/194). Apparently, there are models where "arg1 > 1" and it a different instance of overflow and wraparound of addresses.